src/components/productSection/ProductsSection.jsx

```
@@ -36,7 +36,7 @@ const ProductSection = () => {
                 {[1, 2, 3].map((_, i) => (
                   <img
                     key={i}
-                    src={subImage}
+                    src={thumbIndex === 0 ? product.image : `./images/${product.title.toLowerCase().replace(' ', '-')}-view-${thumbIndex}.png`}
```

ESLint

[error] 'thumbIndex' is not defined.

(no-undef)